spec/services/moves/updater_spec.rb

```diff
@@ -31,6 +31,8 @@
     }
   end
 
+  before { create :supplier_location, supplier: move.supplier, location: move.from_location } # Used by SupplierChooser
```

Comment:
This will soon be redundant as supplier chooser is not called when updating a move. Maybe add a TODO here as a reminder to clean this up if this gets merged before the other PR.

Reply:
good spot!